lib/jeff/acu.ex

```diff
@@ -121,7 +121,7 @@ defmodule Jeff.ACU do
 
         {:ok, bytes} ->
           try do
-            Message.decode(bytes)
+            _ = Message.decode(bytes)
```

Comment:
This looks strange to me, but I don't know the code. The message is decoded and then the result is ignored?

Reply:
I think the logic here is that it decodes to make sure the data received was actually good and not garbled?

I'm not entirely sure TBH